api/api/urls.py

```diff
@@ -15,8 +15,11 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.conf import settings
+from django.urls import path, include
+from django.conf.urls.static import static
 
 urlpatterns = [
     path("admin/", admin.site.urls),
-]
+    path("documents/", include("documents.urls")),
+] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
```

Comment:
**nit(non-blocking):** it’s a good prctice to collect statics and serve them via nginx
https://forum.djangoproject.com/t/configure-static-files-to-work-with-nginx/5689/2